### PR TITLE
removed QImage from libmscore

### PR DESCRIPTION
**Pull request by handrok**
Resolves: *(direct link to the issue)*

*(short description of the changes and the motivation to make the changes)*

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: src/engraving/draw/iimageconverter.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#include "pixmap.h"
+
+namespace mu::draw {
+class IImageConverter : MODULE_EXPORT_INTERFACE
+{
+    INTERFACE_ID(IImageConverter)
+public:
+    virtual ~IImageConverter() = default;
+
+    virtual Pixmap scaled(const Pixmap& origin, const Size& s) = 0;
```
Comment (igorkorsukov) [inline src/engraving/draw/iimageconverter.h:37]: I think, needs to add `const`

Comment (handrok) [reply src/engraving/draw/iimageconverter.h:37]: sure

File: src/engraving/draw/pixmap.h [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    {
+        m_Data = data;
+        m_cashKey = qHash(data);
+    }
+
+    uint cashKey() const { return m_cashKey; }
+
+private:
+    Size m_Size;
+    QByteArray m_Data;
```
Comment (igorkorsukov) [inline src/engraving/draw/pixmap.h:57]: should be `m_size` and `m_data`

Comment (handrok) [reply src/engraving/draw/pixmap.h]: fixed

File: src/engraving/draw/qimageconverter.h [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        QByteArray bytes;
+        QBuffer buffer(&bytes);
+        buffer.open(QIODevice::WriteOnly);
+        qtPixMap.save(&buffer, "PNG");
+
+        Pixmap result({ qtPixMap.width(), qtPixMap.height() });
+        result.setData(bytes);
+
+        return result;
+    }
```
[the remark this answers is not in the document]
Comment (handrok) [reply src/engraving/draw/qimageconverter.h]: fixed

File: src/engraving/draw/pixmap.h [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#ifndef MU_DRAW_PIXMAP_H
+#define MU_DRAW_PIXMAP_H
+
+#include <QByteArray>
+
+#include "modularity/ioc.h"
```
Comment (igorkorsukov) [inline src/engraving/draw/pixmap.h:27]: not needed here

Comment (handrok) [reply src/engraving/draw/pixmap.h]: fixed

**Review by igorkorsukov — APPROVED** [review]

**Pull request merged**